simulariumio/cytosim/cytosim_converter.py

```diff
@@ -71,7 +77,7 @@ def _parse_object_dimensions(
                     agents += 1
                 continue
             if is_fiber:
-                subpoints += 1
+                subpoints += VALUES_PER_3D_POINT
```

Comment:
I'm not sure but it looks like this could be using SUBPOINT_VALUES_PER_ITEM ?

Reply:
yep, fixed it a few other places too